core/src/main/scala/org/apache/pekko/persistence/r2dbc/query/scaladsl/QueryDao.scala

```diff
@@ -179,10 +179,10 @@ private[r2dbc] class QueryDao(settings: R2dbcSettings, connectionFactory: Connec
       row =>
         if (backtracking)
           SerializedJournalRow(
-            slice = row.get("slice", classOf[Integer]),
+            slice = row.get[Integer]("slice", classOf[Integer]),
```

Comment:
Why scala3 can't infer this type info?

Reply:
I don't know. Scala 3 is not Scala 2 and does not work excatly the same as Scala 2. The code does not work in Scala 3 if I do not add this change.

Reply:
Yes, I was expect Scala3 has better type inference.